### ISSUE TITLE
HM Admin Dashboard does not do pagination

### ISSUE THREAD

**Issue by eigood** [moqui/HiveMind#40]
If I go to HM Admin > Projects, it will paginate all projects.  The dashboard will only display in-progress projects.  But, it does *no* pagination.  If there are hundreds to thousands of in-progress projects, it will send that much pre-rendered html from the server, and the browser struggles to display that much content.

I started to create new projects on demo.moqui.org, and never got a paginator to display(but hadn't gotten it up to 10 yet).  The project list tho, was displaying a paginator with only 3 items on it(well below the per-page setting).

**Comment by jonesde** [comment]
Agreed, the HM Admin app needs better support for large numbers of projects (ie more than ~100, HM Admin right now is really designed for more like 10-20 active projects). This is the reason the UI in Marble ERP for projects is quite different, the Marble ERP screens being meant for a large number of projects (and not containing any screens that don't handle a large number of projects).

On a technical note, to display a list of summaries per projects it uses section-iterate (not form-list). Pagination is supported for section-iterate but isn't quite as fancy, that didn't exist when HM Admin was written making paginating those much more cumbersome (can be done manually anywhere), at this point it mostly just needs to be implemented on the couple of screens in HM Admin that list projects without pagination.
